app/views/site/contribute.erb

```diff
@@ -0,0 +1,168 @@
+            We look at all sorts of things: style, idiomatic use of language features, naming, missing test cases, and redundant test cases. We also try to avoid forcing the solution to take a particular approach. Subscribe to the language track repositories of your favorite languages so you get notifications about new issues and pull requests. The track maintainers will greatly appreciate your help!
+          </p>
+
+          <p>
+             You can search for language track repositories <a href="https://github.com/exercism/">here</a>.
+          </p>
+        </div>
+
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Canonicalize Data</h2>
+
+          <p>
+            Different implementations of the same exercise often drift apart as people solving them discover different, interesting edge cases. Defining a canonical set of test data helps improve the quality all the implementations, and makes it easier to port the exercise to new languages.
+          </p>
+
+          <p>
+            Do a survey of the existing implementations, and decide on a good set of test data. Submit a pull request to the <a href="https://github.com/exercism/x-common">metadata repository</a> suggesting a good set of test data, and the track maintainers will be delighted to discuss it with you.
+          </p>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Help Triage Issues</h2>
+
+          <p>
+            We've got a huge number of old issues that aren't particularly actionable, and new issues come in all the time.
+          </p>
+
+          <p>
+            Is an issue no longer relevant? Can it be closed? Do we need more information? If it's a bug, is it reproducible? If it's a discussion and we need to make some sort of decision, then it should be labeled as such.
+          </p>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Fix Bugs</h2>
+
+          <p>
+            Exercism is currently a collection of experiments that grew into an app with the help of duct tape, chewing gum, and chicken wire. It's a mess, and it's got bugs.
+          </p>
+
+          <p>
+            Check out <a href="https://github.com/exercism/exercism.io/issues?q=is%3Aissue+is%3Aopen+label%3Abug">the current issues</a> if you want to give it a try!
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Add Features</h2>
+
+          <p>
+            Exercism is a collective work, we are always open to suggestions and new ideas.
+          </p>
+
+          <p>
+            Have a new idea and want to discuss it? <a href="https://github.com/exercism/discussions/issues">Open a discussion</a> so we can push things further.
```

Comment:
Here we have "discuss" and "discussions" right after each other. Maybe _Have a new idea? [Open a discussion] so we can push things further._